src/routes/radio/+page.svelte

```diff
@@ -1,8 +1,104 @@
-<script>
-  import CommonHead from '$lib/components/CommonHead/CommonHead.svelte';
-  import NotFinished from '$lib/components/NotFinished/NotFinished.svelte';
+<script lang="ts">
   import { m } from '$lib/paraglide/messages';
+  import CommonHead from '$lib/components/CommonHead/CommonHead.svelte';
+  import tireTrack from '$lib/assets/images/original/tire_tracks.jpg';
+  import Player from '$lib/components/Radio/Player.svelte';
+  import { writable } from 'svelte/store';
+
+  const nowPlaying = writable<string | null>('Turbo Groove');
 </script>
 
 <CommonHead>{m['radio.title']()}</CommonHead>
-<NotFinished />
+
+<div class="relative min-h-screen overflow-hidden bg-gray-100 py-8 dark:bg-gray-900">
+  <img
+    src={tireTrack}
+    alt="Tire background"
+    class="pointer-events-none absolute inset-0 h-full w-full object-cover opacity-100 dark:opacity-70"
+  />
+  <div class="pointer-events-none absolute inset-0 bg-black/90 dark:bg-black/70"></div>
+
+  <div class="relative z-10 mx-auto w-full px-3">
+    <Player streamUrl="/stream" stationName="ASEAN Motor Club Radio">
+      <span slot="current">{$nowPlaying}</span>
+    </Player>
+  </div>
+
+  <div
+    class="bg-background/80 dark:bg-background-900/80 relative z-10 mx-auto mt-8 max-w-[700px] rounded-lg border border-gray-300 p-6 text-gray-800 backdrop-blur-sm dark:border-gray-700 dark:text-gray-200"
+  >
+    <h2 class="mb-4 text-2xl font-bold text-gray-900 dark:text-white">Radio Features</h2>
```

Comment:
please add all these static messages to `messages/en.json` (I see you already use with radio player)